api/rosetta/data_transaction_integration_test.go

```diff
@@ -0,0 +1,555 @@
+// Copyright 2021 Optakt Labs OÜ
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+// +build integration
+
+package rosetta_test
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/labstack/echo/v4"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/onflow/flow-go/model/flow"
+
+	"github.com/optakt/flow-dps/api/rosetta"
+	"github.com/optakt/flow-dps/models/dps"
+	"github.com/optakt/flow-dps/rosetta/configuration"
+	"github.com/optakt/flow-dps/rosetta/identifier"
+	"github.com/optakt/flow-dps/rosetta/meta"
+	"github.com/optakt/flow-dps/rosetta/object"
+)
+
+func TestGetTransaction(t *testing.T) {
+
+	db := setupDB(t)
+	api := setupAPI(t, db)
+
+	var (
+		firstHeader      = knownHeaders(44)
+		multipleTxHeader = knownHeaders(165)
+		lastHeader       = knownHeaders(181)
+	)
+
+	const (
+		firstTx = "d5c18baf6c8d11f0693e71dbb951c4856d4f25a456f4d5285a75fd73af39161c"
+
+		// two transactions in a single block
+		firstOfTwoTx  = "23c486cfd54bca7138b519203322327bf46e43a780a237d1c5bb0a82f0a06c1d"
+		secondOfTwoTx = "3d6922d6c6fd161a76cec23b11067f22cac6409a49b28b905989db64f5cb05a5"
```

Comment:
I find it a bit confusing to have `first` twice. How about `oneOfTwo` and `twoOfTwo`?

Reply:
I'd even go as far as suggest making the second test value a slice and using `testTxs[0]` & `testTxs[1]`, this way we use the language to make it explicit rather than naming. WDYT?

Reply:
Not a bad idea @Ullaakut , I did exactly that :)